target/ppc/translate.c

```diff
@@ -3196,6 +3196,21 @@ static inline void gen_align_no_le(DisasContext *ctx)
                       (ctx->opcode & 0x03FF0000) | POWERPC_EXCP_ALIGN_LE);
 }
 
+static bool do_ea_calc(DisasContext *ctx, int ra, TCGv displ, TCGv *ea)
```

Comment:
I think we could declare this function as inline ("static inline bool").

And do we need ea to be passed as a pointer? Wouldn't "TCGv ea" work just fine?